FEATURE_GATES.md

```diff
@@ -75,6 +75,7 @@ Features that reach GA and over time become stable will be removed from this tab
 | KongCustomEntity                        | `false` | Alpha | 3.2.0  | 3.3.0 |
 | KongCustomEntity                        | `true`  | Beta  | 3.3.0  | 3.4.0 |
 | CombinedServicesFromDifferentHTTPRoutes | `false` | Alpha | 3.4.0  | TBD   |
+| StickySessionsTerminatingEndpoints      | `false` | Alpha | TBD    | TBD   |
```

Comment:
We are adding a new feature gate here but it may not fit a patch release. Should we ask the product team for how to enable/disable the feature?

Reply:
this feature will be released with kong-operator